packages/flutter/lib/src/rendering/viewport_offset.dart

```diff
@@ -227,6 +242,15 @@ class _FixedViewportOffset extends ViewportOffset {
     // Do nothing, viewport is fixed.
   }
 
+  @override
+  Future<Null> animateTo(double to, {
+    @required Duration duration,
+    @required Curve curve,
+  }) {
+    // Do nothing, viewport is fixed.
+    return new Future<Null>.value();
+  }
+
```

Comment:
we could implement this more briefly by doing:

`animateTo(...) async => null`